packages/cli/src/commands/start.ts

```diff
@@ -347,8 +348,11 @@ export class Start extends BaseCommand {
 			// @ts-ignore
 			const webhookTunnel = await localtunnel(port, tunnelSettings);
 
-			process.env.WEBHOOK_URL = `${webhookTunnel.url}/`;
-			this.log(`Tunnel URL: ${process.env.WEBHOOK_URL}\n`);
+			// Change required here
+			config.set('endpoints.webhookUrl', webhookTunnel.url);
```

Comment:
shouldn't this be 
```suggestion
			config.set('endpoints.webhookUrl', `${webhookTunnel.url}/`);
```
to be backward compatible?

Reply:
You are correct, I've accepted your change.

Thanks for having a look :)